autonomy/deploy/base.py

```diff
@@ -59,51 +69,130 @@ class NotValidKeysFile(Exception):
     """Raise when provided keys file is not valid."""
 
 
-# TODO: how dpes this relate to `autonomy/services`? Unify
```

Comment:
To answer the question, `ServiceBuilder` is to the `Service` component what `AEABuilder` is to the `Agent` component